Include replicas in package search

replaceafill opened the pull request:
This PR allows to search for a package UUID and get its replicas returned in the packages table. Searching for a replica UUID will also return the replicated package.

Connected to https://github.com/archivematica/Issues/issues/801

ross-spencer reviewed (approved):
This is pretty cool code @replaceafill. Looks great to me. :+1:

replaceafill committed:
Include replicas in package search

This commit allows to search for a package UUID and get its replicas
returned in the packages table. Searching for a replica UUID will also
return the replicated package.
